TimberAPI/AssetLoaderSystem/ResourceAssetPatch/ResourceAssetLoaderTimberApi.cs

```diff
@@ -0,0 +1,80 @@
+using Bindito.Core;
+using HarmonyLib;
+using Timberborn.AssetSystem;
+using TimberbornAPI.AssetLoaderSystem.AssetSystem;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+namespace TimberbornAPI.AssetLoaderSystem.ResourceAssetPatch
+{
+    public class ResourceAssetLoaderTimberApi : IResourceAssetLoader
+    {
+        private readonly IAssetLoader _assetLoader;
+
+        private Shader _shader;
+
+        private Shader Shader
+        {
+            get
+            {
+                if (_shader == null)
+                    _shader = Resources.Load<GameObject>("Buildings/Paths/Platform/Platform.Full.Folktails").GetComponent<MeshRenderer>().materials[0].shader;
+                return _shader;
+            }
+        }
+
+        public ResourceAssetLoaderTimberApi(IAssetLoader assetLoader)
+        {
+            _assetLoader = assetLoader;
+        }
+
+        public T Load<T>(string path) where T : Object
+        {
+            T item = Resources.Load<T>(path);
+
+            if (item != null)
+                return item;
```

Comment:
I know this is opinionated stuff, but I always use braces on if and for blocks, even if they are just 1 liners. IMO it's clearer that way

Reply:
That reminds me that I never did the second round o